### PR TITLE
Feat: Display At Mentions in Activity List

### PR DESCRIPTION
**Pull request by bzang**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "lodash"
   ],
   "presets": [
-    "es2015",
```
Comment (adamweeks) [inline .babelrc]: This should stay, right?

File: packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    if (e.keyCode && e.keyCode === 13) {
+      return triggerEvent(e);
+    }
+    return false;
+  }
+
+  function triggerEvent(e) {
+    const target = e.target;
+    if (isMention(target)) {
+      onEvent(EVENT_NAME_MENTION_CLICKED, {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js]: Should we check if onEvent exists first?

File: packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        onEvent={onEvent}
+      />
+    );
+
+    return Object.assign({}, activity, {component});
+  }
+  return activity;
+};
+
+function constructHydraId(type, id) {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js]: We should eventually move this to a utility function, it is all over the codebase.

File: packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.test.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-nested-callbacks */
+import {
+  formatActivity
+} from './selectors';
+
+describe(`formatActivity function`, () => {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.test.js]: img

File: packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js (hunk 3 above)
Comment (bzang) [reply packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js]: Agreed. I wanted to but didn't want to create another package in this PR for it.

File: .babelrc (hunk 1 above)
Comment (bzang) [reply .babelrc]: Babel is pushing people to the `preset-env` setup and actually warning people when they install the `es2015` preset. That correctly transforms get pulled in based on our build targets.

File: packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js (hunk 2 above)
Comment (bzang) [reply packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js]: It's a bit obtuse but the `onEvent` check is in the `isMention()` check.

**Pull request merged**